Packs/1Password/Integrations/OnePasswordEventCollector/README.md

```diff
@@ -0,0 +1,72 @@
+# 1Password Event Collector
+
+Fetch events about actions performed by 1Password users within a specific account, access and modifications to items in shared vaults, and user sign-in attempts.
+
+This integration was integrated and tested with V2 endpoints of the 1Password Events API.
+
+## Configure 1Password Event Collector in Cortex
+
+The integration can be configured to fetch three types of events from 1Password:
+
+- **Audit events** - Information about actions performed by team members within a 1Password account. Events include when an action was performed and by whom, along with details about the type and object of the action and any other information about the activity.
+
+- **Item usage actions** - Information about items in shared vaults that have been modified, accessed, or used. Events include the name and IP address of the user who accessed the item, when the item was accessed, and the vault where the item is stored.
+  
+- **Sign in attempts** - Information about sign-in attempts. Events include the name and IP address of the user who attempted to sign in to the account, when the attempt was made, and, for failed attempts, the cause of the failure.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Server URL | The API server URL depends on the domain where the account is hosted. Refer to the integration Help section for more details. | True |
+| API Token | The bearer token used to authenticate with the 1Password Events API. This must include the required features (scopes) that correspond to the event types to be fetched. Refer to the integration Help section for more details. | True |
+| Trust any certificate (not secure) | Allow connections without verifying the SSL certificate of the server. | False |
+| Use system proxy settings |  | False |
+| Fetch Events | Whether to fetch events from 1Password. | False |
+| Types of events to fetch | Types of events to fetch from 1Password. Possible values are: Audit events, Item usage actions, Sign in attempts. | True |
+| First fetch time | First event timestamp to fetch. For example: "1 min ago", "2 weeks ago", "3 months ago". Default is "2 weeks ago". | False |
+| Maximum number of audit events per fetch |  | True |
+| Maximum number of item usage actions per fetch |  | True |
+| Maximum number of sign in attempts per fetch |  | True |
+
+## Limitations
+
+- It is recommended to configure the integration instance so that the maximum number of fetched events does not exceed **100,000 per minute per event type**. Otherwise, the 1Password Events API may raise rate limit errors (HTTP 429).
```

Comment:
```suggestion
It is recommended to configure the integration instance so that the maximum number of fetched events does not exceed **100,000 per minute per event type**. Otherwise, the 1Password Events API may raise rate limit errors (HTTP 429).
```

Reply:
Fixed